common/src/main/java/io/netty/util/NetUtil.java

```diff
@@ -266,12 +270,27 @@ public Integer run() {
                             logger.debug("{}: {}", file, somaxconn);
                         }
                     } else {
-                        if (logger.isDebugEnabled()) {
-                            logger.debug("{}: {} (non-existent)", file, somaxconn);
+                        // Try to get from sysctl
+                        Integer tmp = null;
+                        if (SystemPropertyUtil.getBoolean("io.netty.net.somaxconn.trySysctl", false)) {
```

Comment:
@normanmaurer - WDYT ... is defaulting to `false` good enough? Should I just rip this out?